var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -90,6 +90,13 @@ def configure_args(self):
             '-g' if '+debug' in spec else '',
         ]
 
+        config_ldflags = []
+        # PGI runtime libraries
+        if '%pgi' in spec:
+            config_ldflags.append('-pgf90libs')
+        if '+parmetis' in spec:
+            config_ldflags.append('-L{0}'.format(spec['metis'].prefix.lib))
```

Comment:
Why was this line moved up here?

Reply:
Hi Adam,

Scope creep

Hi Adam,

> Why was this line moved up here?

I saw that there were list variables for both `config_args` and `config_cflags` and thought I should follow suite with `config_ldflags`. Then I saw that lone `--with-ldflags` switch for `parmetis` down below and thought that perhaps it was more consistent to treat all those together. I'm happy to retract that bit; I just got carried away.

Reply:
I would probably undo that change if the previous way works

Reply:
Okay, reverted the parmetis change. Now the patch just defines `config_ldflags` and applies the fix for PGI.